beacon-chain/beacon-state-network.md

```diff
@@ -38,17 +38,17 @@ class BeaconStateProof(Container):
     See: https://github.com/ethereum/consensus-specs/blob/dev/ssz/merkle-proofs.md#merkle-multiproofs
     """
     root: Bytes323
-    leave_indices: List[unit64, LEAVE_ITEM_LIMIT]  # GeneralizedIndex is represented by unit64
-    leaves: List[Bytes32, LEAVE_ITEM_LIMIT]
+    leaf_indices: List[unit64, LEAVE_ITEM_LIMIT]  # GeneralizedIndex is represented by unit64
+    leafs: List[Bytes32, LEAVE_ITEM_LIMIT]
```

Comment:
```suggestion
    leaves: List[Bytes32, LEAVE_ITEM_LIMIT]
```

Reply:
good catch I forgot the pural of leaf was leaves 😄

Reply:
resolved